hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/steps/StakePeriodChanges.java

```diff
@@ -137,10 +134,6 @@ public boolean process(
                 logger.error("CATASTROPHIC failure updating end-of-day stakes", e);
                 stack.rollbackFullStack();
             }
-            if (config.getConfigData(TssConfig.class).keyCandidateRoster()) {
-                tssBaseService.regenerateKeyMaterial(stack);
-                startKeyingCandidateRoster(dispatch.handleContext(), newWritableRosterStore(stack, config));
-            }
```

Comment:
nit: This need not be deleted. But since we didn't enable the feature flag, it doesn't matter.

Reply:
Restored to fix unit test. 🙏